### PR TITLE
Story/quad 100 sprint3

### PR DESCRIPTION
**Pull request by MeghanTalasila**
@jdamerow  @charan1312  QUAD-100: Not yet complete. Updated controller.

Work remaining: Fetch data onto jsp and display public projects information.

### CODE REVIEW

**Comment by jdamerow** [comment]
please resolve conflicts

**Comment by MeghanTalasila** [comment]
@jdamerow Story/QUAD-100 conflicts are resolved

File: Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/workbench/RetrieveProjectDAO.java [hunk 1]
```diff
@@ -86,6 +86,29 @@ public ProjectDTO getProjectDTO(String projectId,String userId) throws QuadrigaS
 	}
 
 
+	/**
+	 * Meghana
+	 */
+	@SuppressWarnings("unchecked")
+	@Override
+	public List<ProjectDTO> getAllProjectsDTOByAccessibility(String accessibility) throws QuadrigaStorageException 
+	{
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/workbench/RetrieveProjectDAO.java:95]: please use the code style we use
https://diging.atlassian.net/wiki/pages/viewpage.action?pageId=8781898

Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/workbench/RetrieveProjectDAO.java:95]: javadoc missing

File: Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/workbench/RetrieveProjectManager.java [hunk 2]
```diff
@@ -56,6 +56,18 @@
...
 	/**
+	 * 
+	 */
+	@Override
+	@Transactional
+	public List<IProject> getProjectListByAccessibility(String accessibility) throws QuadrigaStorageException
+	{
+		List<IProject> projectList;
+		projectList =  projectShallowMapper.getProjectList(accessibility);
+		return projectList;
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/workbench/RetrieveProjectManager.java:67]: make this all one line

```
return xxx;
```

Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/workbench/RetrieveProjectManager.java:67]: javadoc missing

Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/workbench/RetrieveProjectManager.java:67]: Also, I'd do that the other way around. Let your manager class get the DTOs and then map them using the mapper. Then you can get rid of the getProjectListByAccesibility method in the mapper classes.

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/HomeController.java [hunk 3]
```diff
@@ -48,6 +55,9 @@
     @Autowired
     private IUserProfileManager profileManager;
 
+	@Autowired 
+	private IRetrieveProjectManager projectManager;
+
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/HomeController.java:60]: delete if not used

**Pull request closed without merging**